### ISSUE TITLE
Re-do Codeception tests not to require a server

### ISSUE THREAD

**Issue by samdark** [yiisoft/app#50]
We can use functional tests only and remove any acceptance ones.

**Comment by terabytesoftw** [comment]
Why eliminate the acceptance tests, if we can even implement webdriver with aerokube selenium and enhance everything later.

**Comment by samdark** [comment]
Yeah, maybe removing them is too much :) I'm more about https://github.com/yiisoft/app/blob/master/tests/functional.suite.yml#L12 this config part that mentions a web server.

**Comment by samdark** [comment]
In order to solve it we need to implement PSR module for Codeception.

**Comment by samdark** [comment]
https://trello.com/c/wIwCQyeN/112-implement-generic-psr-7-module-for-codeception

**Comment by samdark** [comment]
Closing since now there are both functional and acceptance ones.
